docs_user/modules/proc_deploying-backend-services.adoc

```diff
@@ -91,44 +88,40 @@ you use `install_yamls`:
 make input
 ----
 endif::[]
+ifeval::["{build}" == "downstream"]
+. Create the {OpenStackShort} secret. For more information, see link:https://docs.redhat.com/en/documentation/red_hat_openstack_services_on_openshift/{rhos_curr_ver}/html-single/deploying_red_hat_openstack_services_on_openshift/index#proc_providing-secure-access-to-the-RHOSO-services_preparing[Providing secure access to the Red Hat OpenStack Services on OpenShift services] in _Deploying Red Hat OpenStack Services on OpenShift_.
+endif::[]
 
-. If the `$ADMIN_PASSWORD` is different than the already set password
-in `osp-secret`, amend the `AdminPassword` key in the `osp-secret`
-correspondingly:
+. If the `$ADMIN_PASSWORD` is different than the password you set
```

Comment:
Do steps like this typically begin with "Optional:" ?

Reply:
I don't think it would be labeled "optional" because the step is required for those who have a different password from the osp-secret. It's a conditional step.